### ISSUE TITLE
Favicon looks fuzzy in screens that have a higher pixel density

### ISSUE THREAD

**Issue by moodmosaic** [haskell-infra/hl#88]
Here's a possible way to solve this:

>"Old (non-retina) favicons are 16 × 16 px; a retina favicon is thus 32 × 32 px. The lazy way to support retina is to replace your old 16 px favicon.ico file with a 32 px file, and allow non-retina browsers to scale the image. The proper solution, however, is to create a single favicon.ico file containing two icon resources: one 16 × 16, the other 32 × 32. ICO files support other resolutions as well, but I see no practical utility in doing so."

– http://daringfireball.net/2013/01/retina_favicons

**Comment by chrisdone** [comment]
Yeah, someone pointed it out on hacker news. I have a retina screen but chrome doesn't pay attention to my magnification settings so it leaves the icons as 16x16, so I didn't notice this. Thanks for the ticket.

**Comment by moodmosaic** [comment]
Just in case it helps, I've noticed that http://ghcformacosx.github.io/ uses a favicon which shows nice on retina screens.
